docs/content/community/bijeenkomsten/20180802-demo-sprint-2.md

```diff
@@ -0,0 +1,153 @@
+
+
+#### Dit soort demo's is leuk / zinvol
+
+Dank! Het is deel van de agile scrum-methodiek, waaraan we zo goed mogelijk invulling proberen te geven ondanks de uitdagingen qua werken op afstand en de grootte van het team. De demo's zijn erg belangrijk voor de transparantie over de voortgang, het controleren of in de juiste richting wordt gewerkt en om feedback op te halen.
+
+
+#### Kunnen leveranciers ook zelf dingen in demo tonen?
+Dit is iets te bespreken in de bijeenkomst met leveranciers, zie issue [#280](https://github.com/VNG-Realisatie/gemma-zaken/issues/280). We moeten zorgen dat enerzijds het speelveld voor alle leveranciers gelijk blijft, en anderzijds de betrokkenheid zo hoog mogelijk is en blijft. Er zijn allerlei opties, te denken valt bijvoorbeeld aan een demo's door een gemeente die een user story aan heeft gedragen voor uitbreiding van ZDS 2.0 en deze vervolgens intern door een leverancier laat implementeren.
+
+
+#### Na beslissing in de Regiegroep is het een tijd stil geweest, waarom was er geen communicatie?
+
+Punt is helder en doorgegeven aan VNG Realisatie.
+
+
+#### Gaan we eisen (architectuur) dat alles zaaksystemen met componenten van anderen kunnen werken?
+
+Ja, dat is wel expliciet de bedoeling. We standaardiseren op het niveau van de API specificaties. In de bijbehorende architctuur en afspraken zal o.a. staan dat componenten iedere los gedefiniëerde API ook los moeten aanbieden. Een gevolg zal zijn dat zaaksystemen met componenten van anderen kunnen werken. Dit maakt een einde aan de huidige praktijk waarbij functionaliteit vaak onnodig redundant in organisaties aanwezig is, en waarbij data gekopiëerd en gesynchroniseerd moet worden.
+
+#### Is StUF end-of-life?
+
+Nee, de transitie naar nieuwe standaarden kan niet in de vorm van een Big Bang. Er zal een lange periode zijn waarbij het nieuwe naast het oude bestaat. StUF blijft van groot belang. Wel is het zo dat gestreeft wordt naar een gegevenslandsschap, wat praktisch gezien alleen vorm kan krijgen wanneer we op de nieuw manier standaardiseren.
+
+
+#### Zijn er naast ZDS nog meer standaarden in ontwikkeling op deze manier, en weten de verschillende initiatieven van elkaar?
+
+Het ZDS 2.0 traject is het meest concreet aan het werk met een team, projectplan en budget. Daarnaast wordt gewerkt aan API's voor RSGB, qua proces op een andere manier. Verschillende betrokkenen bij ZDS 2.0 zijn ook daar bij betrokken. Nadere afstemming (ook over proces van realisatie) vindt voortdurend plaats.
+
+
+### Inhoudelijk
+
+#### Waar kunnen we de ‘zoemende’ API’s vinden?
+
+|Specificatie|Referentie|
+|---|---|
+|[ZRC OAS](https://ref.tst.vng.cloud/zrc/api/v1/schema/)|[zoemende API](https://ref.tst.vng.cloud/zrc/api/v1/)|
+|[ZTC OAS](https://ref.tst.vng.cloud/ztc/api/v1/schema/)|[zoemende API](https://ref.tst.vng.cloud/zrc/api/v1/)|
+|[DRC OAS](https://ref.tst.vng.cloud/ztc/api/v1/schema/)|[zoemende API](https://ref.tst.vng.cloud/drc/api/v1/)|
+
```

Comment:
kunnen we hier ook linken naar https://ref.tst.vng.cloud/developers/ en eventueel deze docs uitbreiden met de shortcuts?